### PR TITLE
Acceptance test VM bootstrap improvements

### PR DESCRIPTION
**Pull request by purbon**
* Add methods to destroy, bootstrap and halt a list of machines either all of them or the ones listed under a given platform name.

* Also refactor the rake task namespace to be under qa and not the generic
test.

Complementary to #5218

### CODE REVIEW

File: qa/Rakefile [hunk 1]
```diff
@@ -9,19 +9,42 @@ platforms = PlatformConfig.new
...
+    desc "Bootstrap all the VM's used for this tests"
+    task :setup, :platform do |t, args|
+      config   = PlatformConfig.new
+      machines = config.select_names_for(args[:platform])
+
+      message  = "bootstraping all VM's defined in acceptance/Vagrantfile"
+      message  = "#{message} for #{args[:platform]}: #{machines}" if !args[:platform].nil?
+      puts message
+
+      LogStash::VagrantHelpers.destroy   machines
```
Comment (ph) [inline qa/Rakefile]: many spaces between destroy and machine and should we use parentheses for theses methods calls?

**Comment by ph** [comment]
+1 for the namespace.

When I run

```
bundle exec rake qa:vm:setup\[redhat\]
```
I get this output:

```
bootstraping all VM's defined in acceptance/Vagrantfile for redhat: []
```
is the empty `[]` expected?

**Comment by purbon** [comment]
this happen because in this PR, redhat based machines are still named centos as type, this naming is fixed in https://github.com/elastic/logstash/pull/5226

**Comment by purbon** [comment]
see 
```
skywalker% bundle exec rake qa:vm:setup["centos"]
bootstraping all VM's defined in acceptance/Vagrantfile for centos: ["centos-6", "centos-7"]
```

**Comment by ph** [comment]
LGTM

**Pull request closed without merging**